docs/1 Introduction/1 Introduction.md

```diff
@@ -1,45 +1,56 @@
 # An introduction to Starport
 
-Let us dive into Starport, what we can achieve with it, and which other technologies play well with it.
+Starport is a developer-friendly interface to the Cosmos SDK.
 
 ## What is Starport?
 
-Starport is a tool that makes it easier to create blockchains.
+The Starport tool is the easiest way to create blockchains using the Tendermint Consensus engine and the Cosmos SDK. <!-- @fadeev does it matter that we use Go? --> Starport is written in the Go programming language. The new blockchain has a Proof-of-Stake (PoS) system with validators that can be defined in the genesis block.
```

Comment:
> The new blockchain has a Proof-of-Stake (PoS) system with validators that can be defined in the genesis block.

This shouldn't be in the intro doc.

> does it matter that we use Go?

I would mention that Cosmos SDK modules are written and Go, so is Starport itself. This may be important, because if a developer knows a language, it's easier for them to see themselves using this tool.